Whiskey/Tasks/Invoke-WhiskeyProcess.ps1

```diff
@@ -0,0 +1,118 @@
+function Invoke-WhiskeyProcess
+{
+    <#
+    .SYNOPSIS
+    Starts a process with given arguments
+    
+    .DESCRIPTION

```

Comment:
Change to:

```
The `Process` task runs a process/executable. Specify the path to the executable to run with the task's `Path` property. The `Path` can be the name of an executable that can be found in the `PATH` environment variable, a path relative to your `whiskey.yml` file's directory, or an absolute path.

The task will fail if the process returns a non-zero exit code. Use the `SuccessExitCode` property to configure the task to interpret other exit codes as "success". 

Pass arguments to the process via the `Argument` property. The `Process` task uses PowerShell's `Start-Process` cmdlet to run the process, so that arguments will be passes as-is, with no escaping. YAML strings, however, are usually single-quoted (e.g. `'Value'`) or double-quoted (e.g. `"Value"`). If you're using a single quoted string and need to insert a single quote, escape it by using two single quotes, e.g. `'escape: '''` is converted to `escape '`. If you're using a double-quoted string and need to insert a double quote, escape it with `\`, e.g. `"escape: \""` is converted to `escape: "`. YAML supports other escape sequences in double-quoted strings. The full list of escape sequences is in the [YAML specification](http://yaml.org/spec/current.html#escaping in double quoted style/).

By default, the executable is run from your `whiskey.yml` file's directory (i.e. the build root). Change the working directory with the `WorkingDirectory` property.

# Properties

* `Path` (*mandatory*): the path to the executable to run. This can be the name of an executable if it is in your PATH environment variable, a path relative to the `whiskey.yml` file, or an absolute path.
* `Argument`: a list of arguments to pass to the executable. Read the documentation above for notes on how to properly escape arguments.
* `WorkingDirectory`: the directory the executable will run in/from. By default, this is the build root, i.e. the `whiskey.yml` file's directory.
* `SuccessExitCode`: a list of exit codes that the `Process` task should interpret to mean the process exited successfully. The default is `0`.

# Examples

## Example 1

        BuildTasks:
        - Process:
            Path: cmd.exe
            Argument:
            - /C
            - dir C:\

This example demonstrates how to call an executable whose arguments have to be quoted a specific way. In this case, we're using `cmd.exe` to get a directory listing of the `C:\` directory. This example will run `cmd.exe /C dir C:\.

## Example 2

        BuildTasks:
        - Process:
            Path: robocopy.exe
            Argument:
            - C:\Source
            - C:\Desitination
            - /MIR    
            SuccessExitCode:
            - 0
            - 1
            - 2
            - 3
            - 4
            - 5
            - 6
            - 7

This example demonstrates how to configure the `Process` task to fail when an executable can return multiple success exit codes. In this case, `robocopy.exe` can return any value less than 8 to report a successful copy.
```